modules/java-modules/http/src/main/java/org/syslog_ng/http/HTTPDestinationOptions.java

```diff
@@ -58,21 +60,30 @@ public Option getOption(String optionName) {
     return options.get(optionName);
   }
 
-  public String getURL() {
-    return options.get(URL).getValue();
+  public TemplateOption getURL() {
+    return options.getTemplateOption(URL);
+  }
+
+  public TemplateOption getMessageTemplate() {
+    return options.getTemplateOption(TEMPLATE);
   }
 
   public String getMethod() {
-     return options.get(METHOD).getValue();
+    return options.get(METHOD).getValue();
   }
 
   private void fillOptions() {
     fillStringOptions();
+    fillTemplateOptions();
   }
 
   private void fillStringOptions() {
-		options.put(new RequiredOptionDecorator(new StringOption(owner, URL)));
-		options.put(new EnumOptionDecorator(new StringOption(owner, METHOD, METHOD_DEFAULT), HTTP_METHODS));
+    //options.put(new RequiredOptionDecorator(new StringOption(owner, URL)));
```

Comment:
Could you remove this comment? (In general, it is confusing/disturbing when a comment message is actually an unused code.)